### ISSUE TITLE
Import .sqlite data file from xdrip into Juggluco

### ISSUE THREAD

**Issue by Electrik-rich546456** [j-kaltes/Juggluco#114]
Potential feature request.
Is it possible to Import .sqlite BG history data file from xdrip into Juggluco ?

**Comment by j-kaltes** [comment]
No, Juggluco only receives data directly from Freestyle Libre sensors, NFC NovoPens or other instances of Juggluco or Juggluco server.

**Comment by Electrik-rich546456** [comment]
why not ? :-)
would you mind adding that feature please.
